packages/dhis-api/src/DhisApi.js

```diff
@@ -132,20 +133,32 @@ export class DhisApi {
   }
 
   async post(endpoint, data, queryParameters) {
+    if (!this.serverSyncEnable) {
+      throw new Error(`Attempted to push data to ${this.serverName} that should not be pushed`);
+    }
```

Comment:
@edmofro Just want to run this through you.

@IgorNadj and I have discussed and we decided to add this in `DhisApi` as the final gate to prevent any changes go into Laos DHIS2 server. Basically we'll just need to add `serverSyncEnable` env variable for Laos EOC DHIS2.

Ideally it should never reach here for Laos DHIS2 because:
- We will have an option to disable syncing when importing NEW entities by this task (https://github.com/beyondessential/tupaia-backlog/issues/2301)
- We already disabled pushing for all EXISTING Laos entities here (https://github.com/beyondessential/tupaia/pull/2130)

So we put this as the last guard. This is just us being very risk adverse. What do you reckon?

Reply:
Yeah I think it's a good idea! Will review